### DISCUSSION TITLE
Too much mods (1.18.1)

### DISCUSSION THREAD

**Discussion by byStarmont** [TheUsefulLists/UsefulMods#304] [Q&A] [answered]
How many mods should I use? Everyone? Just a few? Should iv´e try combinations? Gimme tips pls (i want to make an Aternos server whit mods for 1.18.1)

**Comment by Fleepe** [comment, marked as answer]
Don't install small insignificant mods like Fast Bench/Furnace. They do too little and create a mess in the mods folder. Check which mods are only suitable for the client - they are not needed on the server, but may create some problems if  you install them on it. Do not install mods aimed at optimizing the network; I personally do not install them, they did practically nothing for me. Mods I recommend for **_1.18.2_**:
[Better Chunk Loading](https://www.curseforge.com/minecraft/mc-mods/better-chunk-loading-forge-fabric)
[Connectivity](https://www.curseforge.com/minecraft/mc-mods/connectivity)
[Chunk Sending](https://www.curseforge.com/minecraft/mc-mods/chunk-sending-forge-fabric)
[Clumps](https://www.curseforge.com/minecraft/mc-mods/clumps)
[FerriteCore](https://www.curseforge.com/minecraft/mc-mods/ferritecore-fabric)
[Ksyxis](https://www.curseforge.com/minecraft/mc-mods/ksyxis)
[LazyDFU](https://www.curseforge.com/minecraft/mc-mods/lazydfu)
[Smooth Chunk Save](https://www.curseforge.com/minecraft/mc-mods/smooth-chunk-save)
These three mods are must have for servers (Starlight/Phosphor are obsolete in 1.20> versions) 
[Lithium](https://modrinth.com/mod/lithium)
[ModernFix](https://www.curseforge.com/minecraft/mc-mods/modernfix)
[Starlight](https://www.curseforge.com/minecraft/mc-mods/starlight)

**Reply by byStarmont** [reply]
Tyvm 4 the answer, but I have another question.

When I install "ModernFix", the game gives me an error message saying "FerriteCore is not installed, it may cause very high RAM usage", but FerriteCore is only for Fabric and NeoForge. Should I remove ModernFix or deal with the high RAM usage?  (and how can i deactivate the error message in the mod config?)

**Reply by shizotoaster** [reply]
Ferritecore has Forge versions for 1.18.2-1.20.1

**Reply by byStarmont** [reply]
I was looking in the wrong place, my bad. But now the game crash when i install FerriteCore and Modern fix
